docs/docs/amoro.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Apache Amoro"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Apache Amoro With Iceberg
+
+**[Apache Amoro(incubating)](https://amoro.apache.org)** is a Lakehouse management system built on open data lake formats. Working with compute engines including Flink, Spark, and Trino, Amoro brings pluggable and
```

Comment:
Does "pluggable and" means pluggable and Table Maintenance/AMS?

Reply:
Includes pluggable monitoring plugins and table maintenance merge scheduling strategy.Cann‘t list them all in doc.

Reply:
It reads strange having "pluggable" side-by-side with "Table Maintenance". Why not using the words from [Amoro README](https://github.com/apache/amoro/blob/master/README.md)

> Amoro brings pluggable and self-managed features for Lakehouse to provide out-of-the-box data warehouse experience ...

Reply:
@czy006 please check this comment.